packages/manager/apps/web/client/app/email-domain/upgrade/upgrade.html

```diff
@@ -0,0 +1,78 @@
+<div data-ovh-alert="{{ $ctrl.alerts.main }}"></div>
+
+<div class="container-fluid">
+    <oui-back-button data-on-click="$ctrl.goBack()">
+        <span data-translate="email_domain_upgrade_title"></span>
+    </oui-back-button>
+
+    <oui-stepper>
+
+        <oui-step-form data-header="{{ ::'email_domain_upgrade_step_header_offer' | translate }}"
+            data-editable="!$ctrl.loading.upgrading" data-valid="$ctrl.model.offer"
+            data-prevent-next="!$ctrl.model.offer"
+            data-on-focus="$ctrl.displaySelectedOffer = false"
+            data-on-submit="$ctrl.displaySelectedOffer = true">
+            <p data-translate="email_domain_upgrade_step_header_offer_description"></p>
+            <div class="row">
+                <oui-select-picker
+                    data-ng-if="$ctrl.displaySelectedOffer"
+                    class="mb-3 col-md-4"
+                    data-label="{{ $ctrl.model.offer.displayName }}"
+                    name="{{:: $ctrl.model.offer.name }}"
+                    data-model="$ctrl.model.offer"
+                    data-values="[$ctrl.model.offer]"
+                    data-variant="light">
+                    <oui-select-picker-footer>
+                        <ovh-manager-order-catalog-price
+                            data-user="$ctrl.user"
+                            data-price="$ctrl.model.offer.duration.prices.withoutTax.value * 100000000"
```

Comment:
it's a shame that order-catalog-price does not take a "price" from agora as a parameter and handle those things internally

Reply:
This was done because of emergency, it will be refactored

Reply:
https://github.com/ovh/manager/pull/2013